# Solicitud
string visible_pair
uint64 timestamp
bool store
---
# Respuesta
bool success
